resources/views/auth/passwords/reset.blade.php

```diff
@@ -1,70 +1,72 @@
-
-                        <div class="form-group row">
-                            <label for="password-confirm" class="col-md-4 col-form-label text-md-right">{{ __('Confirm Password') }}</label>
-                            <div class="col-md-6">
-                                <input id="password-confirm" type="password" class="form-control{{ $errors->has('password_confirmation') ? ' is-invalid' : '' }}" name="password_confirmation" required>
-
-                                @if ($errors->has('password_confirmation'))
-                                    <span class="invalid-feedback">
-                                        <strong>{{ $errors->first('password_confirmation') }}</strong>
-                                    </span>
-                                @endif
-                            </div>
-                        </div>
-
-                        <div class="form-group row mb-0">
-                            <div class="col-md-6 offset-md-4">
-                                <button type="submit" class="btn btn-primary">
-                                    {{ __('Reset Password') }}
-                                </button>
-                            </div>
-                        </div>
-                    </form>
-                </div>
+<div class="container mt-5">
+  <div class="row">
+    <div class="col">
+    </div>
+    <div class="col-6">
+      <div align="center">
+        <img class="p-5" src="/img/md-blue-logo.png">
+      </div>
+      <div align="center">
+        <p class="header1">{{__('Forgot Your Password?')}}</p>
+        <p class="header2">{{__("Enter your email address and we'll send you a reset link.")}}</p>
+      </div>
+      <div align="center">
+        <form method="POST" action="{{ route('password.request') }}" class="bg-light p-4">
+          <div class="form-group">
+            <input type="hidden" name="token" value="{{ $token }}">
+            <label class="input-label">{{ __('Email Address') }}</label>
+            <div class="form-group">
+                <input id="email" type="email" class="form-control{{ $errors->has('email') ? ' is-invalid' : '' }}" name="email" value="{{ $email or old('email') }}" required autofocus>
+                @if ($errors->has('email'))
+                    <span class="invalid-feedback">
+                        <strong>{{ $errors->first('email') }}</strong>
+                    </span>
+                @endif
             </div>
-        </div>
+            <button type="submit" class="btn btn-secondary btn-block mt-3 mb-3 text-light text-uppercase">
+              {{ __('REQUEST RESET LINK') }}
+            </button>
+            <a class=" text-primary input-link" href="{{ route('login') }}">
+              {{ __('Back to Login') }}
+            </a>
+          </div>
+        </form>
+      </div>
+    </div>
+    <div class="col">
     </div>
+  </div>
 </div>
 @endsection
+
+@section('css')
+<style lang='scss' scoped>
```

Comment:
This is in a blade template, not a vue component.  Scoped is not available here, lang is not allowed.

Ideally, this is put into a sep scss file called auth/reset.scss and then uses webpack to build.  Then this css section uses a <link> element to bring in that css.

Because it's not scoped, it's important that you prefix these items with an id that scopes these classes.  So something like #reset-password img, for example.